weather-app/helpers/geoWeather.js

```diff
@@ -0,0 +1,34 @@
+const request = require('request')
+
+const getGeoWeather = (latiTude, longiTude, callback) => {
+    const url = `https://api.darksky.net/forecast/4e01d7a1e991e293fa34f0566b62c80d/${longiTude},${latiTude}`
+
+    request(url, (error, response) => {
+        if (response === undefined) {
+            callback(error, null)
+        }
+
+        const jsonData = JSON.parse(response.body)
+
+        if (!jsonData.daily) {
+            callback(error, null)
```

Comment:
This won't work because at this point you don't have variable `error` because you get `response` from `request`